cmd/export.go

```diff
@@ -27,6 +28,7 @@ var includeContainers bool
 var orgs []uint
 
 func init() {
+	exportCmd.Flags().StringVar(&config, "config", "", "Location of configuration file")
```

Comment:
That's a nice approach. I'd like it to be a bit more explicit on the user interface level, that we for example can either provide a config or specify it as a set of args. This way we could probably not mix them together with the options. What do you think?

Reply:
Sorry. I didn't catch your point... Can you give me some examples?